plugins/sqlfluff-templater-dbt/test/fixtures/dbt/dbt_project/snapshots/issue_1771.sql.after

```diff
@@ -1,6 +1,6 @@
 {% snapshot dim_aggregated_brand_hierarchy_snapshot %}
 
-{{
+    {{
```

Comment:
This is the only slight strangeness in the new algorithm, is that it wants to indent this tag because it's within the snapshot, but the tag itself has a newline within it and it's not smart enough to add newlines and indents *inside* the jinja tag.

Any ideas on good solutions here?

Reply:
good plan. I've updated the code and added a test case which does exactly that.